Makefile

```diff
@@ -7,40 +7,48 @@ PYFILES = $(shell ls yeat/*.py yeat/*/*.py)
 help: Makefile
 	@sed -n 's/^## //p' Makefile
 
-## test:        run automated test suite
+## test:        run only short-running automated tests
 test:
-	pytest --cov=yeat -m 'not long and not bandage and not hifi and not nano'
+	pytest --cov=yeat -m short
 
 ## testlong:    run only long-running automated tests
 testlong:
-	pytest --cov=yeat -m 'long and not bandage and not hifi'
+	pytest --cov=yeat -m long
```

Comment:
Added a bunch of pytest marks to simplify the input for `pytest -m`. Instead of having a long conditional `not` and `not` and so forth, we can just condense it down to one pytest mark.